certora/specs/Delegation.spec

```diff
@@ -215,3 +214,33 @@ rule delegatingWithSigUpdatesVotingPower(env e, DelegationToken.Delegation deleg
         assert delegatedVotingPower(delegation.delegatee) == delegatedVotingPowerBefore + balanceOf(delegator);
     }
 }
+
+// Check that the updated voting power of a delegatee after a transfer is lesser than or equal to the total supply of tokens.
+rule updatedDelegatedVPLTEqTotalSupply(address from, address to) {
```

Comment:
actually I don't understand this rule: we don't query anything after the delegate? and where is that transfer?

Reply:
> I think we should remove the mention to transfer, as it's not needed for this rule.

It would be fixed in #108